src/Npgsql/NpgsqlCommand.cs

```diff
@@ -1352,6 +1352,7 @@ internal async ValueTask<NpgsqlDataReader> ExecuteReader(CommandBehavior behavio
                         if (Log.IsEnabled(NpgsqlLogLevel.Debug))
                             LogCommand(connector);
                         NpgsqlEventSource.Log.CommandStart(CommandText);
+                        connector.CommandStart(CommandText);
```

Comment:
We'll also want to support multiplexing, where there's no connector at this phase. We can have the activity on the NpgsqlCommand instead of on the connector, what do you think?

Reply:
We definitely should support multiplexing. As for having the activity on the NpgsqlCommand - sounds like a hack to me. But it's not like we have better ideas...

Reply:
I'm not so sure... At the end of the day the activity really does correspond to the execution of the command... It seems even a bit cleaner to manage it there...